benchmarks/benchmarks/bench_ufunc.py

```diff
@@ -148,3 +148,62 @@ def time_add_scalar_conv(self):
 
     def time_add_scalar_conv_complex(self):
         (self.y + self.z)
+
+
+class ArgParsing(Benchmark):
```

Comment:
I'd like these in a separate PR, simply because the hosted asv runs nightly, so would not produce a before / after if they get merged together.

Reply:
Good point, see #11453